sh_legshoot.lua

```diff
@@ -0,0 +1,28 @@
+PLUGIN.name = "Legs wyjebnik"
+PLUGIN.author = "Lechu2375"
+PLUGIN.description = "Now hitting right in the legs causes injures."
```

Comment:
You should describe how to customize the plugin (e.g how to add ignored factions) in `PLUGIN.readme`